e2e-tests/e2e-fixtures/mock-gef-facilities.js

```diff
@@ -39,7 +39,7 @@ const anUnissuedCashFacility = ({ facilityEndDateEnabled = false } = {}) => ({
     'Other',
   ],
   detailsOther: 'Other',
-  currency: 'GBP',
+  currency: { id: 'GBP' },
```

Comment:
I checked the Dev database & GEF facilities should look like this. Don't think it will make a difference, but was making consistent with tfm e2e test fixture